package/com.unity.formats.usd/Tests/Editor/EditorTests.cs

```diff
@@ -4,34 +4,42 @@
 using UnityEngine;
 using Scene = USD.NET.Scene;
 using Assert = UnityEngine.Assertions.Assert;
+using System.Collections;
 
 namespace Unity.Formats.USD.Tests
 {
     public class FBXHandednessModeTests
     {
         const string fbxGUID = "86a597c63449d2541b7587ff90e75d91"; // GUID of withCamera.fbx
         const string usdGUID = "f377c4260fb216d4dbe2f6e4d67091b5"; // GUID of withCamera.usd
+        const string leftHandedUsdGUID = "25388fbb44b88c448a40ef8eeb0c3ca2"; // GUID of withCamera_leftHanded.usd
```

Comment:
If this class were a ScriptableObject, you could actually just bind these assets, rather than hard-coding the GUID, I think (I have not tried this, it was just a passing thought)

Reply:
cool it worked, thanks!